src/test/llm/visual_language_model/complete_flow_test.cpp

```diff
@@ -153,7 +163,44 @@ TEST_F(VLMServableFlowTest, unaryBasic) {
     EXPECT_STREQ(parsedResponse["object"].GetString(), "chat.completion");
 }
 
-TEST_F(VLMServableFlowTest, streamBasic) {
+// Only image input is accepted, but expected output can't be predicted
```

Comment:
do we have tests for multiple images and ensuring that order is correct?

Reply:
see changes made in http_openai_handler_test